packages/ui-composer/src/MessageFooterCallout/MessageFooterCalloutAction.tsx

```diff
@@ -6,7 +6,8 @@ const MessageFooterCalloutAction = forwardRef<HTMLButtonElement, ComponentProps<
 	props,
 	ref,
 ): ReactElement {
-	return <Button mi={4} ref={ref} primary small flexShrink={0} {...props} />;
+	const modifiedProps = props.danger ? { ...props, primary: false } : props;
+	return <Button mi={4} ref={ref} primary small flexShrink={0} {...modifiedProps} />;
```

Comment:
Could you check if we really need to do this override within this generic component? My understanding is that since `props.danger` is added after primary, it'd take precedence over primary and the button would be displayed as red.

Reply:
Just tested this locally with the old code and just "danger" prop set and found it working as expected

Reply:
It didn’t worked out for me so that’s why I tried this one.

Reply:
That's odd! Let's debug this on our call today